crates/uv-interpreter/src/platform.rs

```diff
@@ -131,7 +131,7 @@ impl FromStr for Arch {
         match s.to_lowercase().as_str() {
             "aarch64" | "arm64" => Ok(Self::Aarch64),
             "armv6l" => Ok(Self::Armv6L),
-            "armv7l" => Ok(Self::Armv7L),
+            "armv7l" | "arm" => Ok(Self::Armv7L),
```

Comment:
This needs to be revisited more holistically; I'm just trying to fix the failure on ARM for now. But, anyway, Rust's `std::env::consts::ARCH` just returns `"arm"`, whereas we need to differentiate between ARM v6 and ARM v7 I guess? I'm also not sure whether the enum variants are intended to match some Rust or Python convention somewhere.This needs to be revisited more holistically; I'm just trying to fix the failure on ARM for now. But, anyway, Rust's `std::env::consts::ARCH` just returns `"arm"`, whereas we need to differentiate between ARM v6 and ARM v7 I guess? I'm also not sure whether the enum variants are intended to match some Rust or Python convention somewhere.